.ignore

```diff
@@ -0,0 +1,21 @@
+dist
```

Comment:
This prevents these files from getting into our search results. Could be part of a different PR. Doesn't effect anything at runtime.